### PR TITLE
feat(grafana): Add HTTP query parameter in DORA dashboard panels

### PR DESCRIPTION
**Pull request by hayk96**
### Summary
This PR introduces a new HTTP query parameter (`var-project=${project:raw}`) to the links within the DORA dashboard's panels. This enhancement ensures that when a user clicks on a panel link, they are directly navigated to the external dashboard that corresponds with the `Search` project, as illustrated in the attached screenshot.

### Screenshots
image

### Other Information
There are some Grafana limitations e.g.: in case of multiple projects the link will not work properly, but it's very helpful when a single project is selected.

### CODE REVIEW

**Comment by Startrekzky** [comment]
Hi @hayk96 , thanks for the PR. Perhaps we should add the time filter variable to the link as well.

File: grafana/dashboards/DORA.json [hunk 1]
```diff
@@ -184,7 +184,7 @@
                   {
                     "targetBlank": false,
                     "title": "",
-                    "url": "/d/${__data.fields[\"metric_hidden\"]}"
+                    "url": "/d/${__data.fields[\"metric_hidden\"]}?var-project=${project:raw}"
```
Comment (Startrekzky) [inline grafana/dashboards/DORA.json:187]: Since the project's variable is added, can you add the time filter variable as well?

**Comment by Startrekzky** [comment]
@hayk96  Also, can you amend your commit message following the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) guidelines?

**Comment by hayk96** [comment]
thanks @Startrekzky! Closing this PR, will open another one.

**Pull request closed without merging**